converters/google/mappers.go

```diff
@@ -152,12 +152,16 @@ func mappers() map[string][]mapper {
 			{
 				convert:           converter.GetBucketIamBindingCaiObject,
 				mergeCreateUpdate: converter.MergeBucketIamBinding,
+				mergeDelete:       converter.MergeBucketIamBindingDelete,
+				fetch:             converter.FetchBucketIamPolicy,
 			},
 		},
 		"google_storage_bucket_iam_member": {
 			{
 				convert:           converter.GetBucketIamMemberCaiObject,
 				mergeCreateUpdate: converter.MergeBucketIamMember,
+				mergeDelete:       converter.MergeBucketIamMemberDelete,
```

Comment:
Is there a reason to keep mergeCreateUpdate and mergeDelete separate? I think of the "merge" action as necessarily including both

Reply:
Yes - we do the merge process in three phases.

1. Fetch full IAM policy from GCP.
2. Process all deleted terraform resources for that policy.
3. Process all added / updated terraform resources for that policy.

We could instead have a single merge function that took a list of changes related to a particular iam policy and split the logic internally so developers didn't have to worry about it, but that would require a significant refactor of terraform validator internals. I'm not sure whether it would be an improvement over the current code or not; there are some things I like about the idea. But either way, it's outside of the scope of this ticket.